velox/duckdb/conversion/DuckConversion.cpp

```diff
@@ -96,8 +96,12 @@ TypePtr toVeloxType(LogicalType type) {
       return BIGINT();
     case LogicalTypeId::FLOAT:
       return REAL();
-    case LogicalTypeId::HUGEINT:
```

Comment:
I wonder if `HUGEINT` can be decimal as well.

Reply:
Is it okay if I handle this case when it is required?

Reply:
that works. Probably open an issue?

Reply:
no need to file an issue. Velox has no BIGINT.